packages/edit-site/src/components/page-pages/index.js

```diff
@@ -129,13 +136,20 @@ export default function PagePages() {
 						</a>
 					);
 				},
+				setList: authors,
+				filters: {
+					in: 'author',
+					notIn: 'author_exclude',
```

Comment:
I think we should keep things simpler at first with just an inclusion filter and probably not even multiple authors(🤔 ). This reveals more nuances about the rest API handling like the priority of filtering based on params - what happens if we have the same value both in `include` and in `exclude`. Probably though we should filter out the active values from each other.